src/main/webapp/app/exercises/programming/assess/code-editor-tutor-assessment-inline-feedback.component.html

```diff
@@ -5,21 +5,41 @@
     [class.alert-danger]="feedback.credits! < 0"
     [class.alert-warning]="!feedback.credits"
 >
-    <div class="m-2">
-        <div *ngIf="!viewOnly; else readOnlyFeedback" class="row align-items-start" (drop)="updateFeedbackOnDrop($event)" (dragover)="$event.preventDefault()">
+    <div *ngIf="!viewOnly; else readOnlyFeedback" class="m-2">
+        <div class="row align-items-start">
             <div class="form-group col-8 p-2">
-                <label jhiTranslate="artemisApp.assessment.detail.feedback"></label>
-                <textarea class="form-control" rows="5" [(ngModel)]="feedback.detailText" [readOnly]="viewOnly"></textarea>
+                <label class="d-inline" jhiTranslate="artemisApp.assessment.detail.feedback"></label>
+                <div class="d-inline" *ngIf="feedback.gradingInstruction!">
+                    <fa-icon [icon]="'question-circle'" class="text-secondary" [ngbTooltip]="'artemisApp.assessment.feedbackHint' | artemisTranslate" container="body"></fa-icon>
+                    <span>{{ feedback.gradingInstruction!.feedback }}</span>
+                </div>
             </div>
             <div class="form-group col-3 p-2">
                 <label jhiTranslate="artemisApp.exercise.score"></label>
-                <input class="form-control px-1" type="number" step="0.5" [(ngModel)]="feedback.credits" [readOnly]="viewOnly || disableEditScore" />
             </div>
             <div class="form-group col-1 p-2" *ngIf="feedback.gradingInstruction!">
                 <jhi-grading-instruction-link-icon [assessment]="feedback"></jhi-grading-instruction-link-icon>
             </div>
         </div>
-        <div *ngIf="!viewOnly">
+        <div class="row align-items-start" (drop)="updateFeedbackOnDrop($event)" (dragover)="$event.preventDefault()">
```

Comment:
Just a question: Whats the rationale behind only having the text field droppable now instead of the entire comment?

Reply:
maybe whole editor-box can be droppable